posts/how-to-install-memgraph-and-memgraph-lab-with-docker-on-macOS.md

```diff
@@ -0,0 +1,233 @@
+
+[Memgraph Lab](https://memgraph.com/product/lab/) is a lightweight and intuitive Cypher and [Bolt](https://boltprotocol.org/) compatible integrated development environment (IDE). It's designed to help you import data, develop, debug, and profile database queries and visualize query results.
+
+In this tutorial, you will install both Memgraph and Memgraph Lab on your Mac. You will then test each installation by running basic Cypher queries to make sure that everything is working correctly.
+
+## Prerequisites
+
+For a seamless installation of Memgraph and Memgraph Lab on your Mac, ensure that you have:
+* A computer running macOS version 10.13 (High Sierra) or higher.
+* Administrative rights to your macOS and an internet connection.
+* A [Docker Hub](https://hub.docker.com/) account, so that you can download the [Docker Desktop for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac) version 1.12 or higher and install it on your computer.
+* Basic knowledge of working with the command line.
+
+## Step 1 — Downloading and Installing Memgraph
+
+You can install Memgraph on macOS using the [Docker](https://www.docker.com/) image provided on Memgraph's website. [Download and save it](https://memgraph.com/download/memgraph/v0.14.1/memgraph-0.14.1-docker.tar.gz) to a location you can quickly locate later on.
+
+Before we go on, it's essential that you [have Docker installed on your system](https://docs.docker.com/docker-for-mac/install/) so you are able to install and run Memgraph.
+
+Once you have Docker installed and the Memgraph Docker image downloaded, run the command below in your command line to begin the installation. Simply replace the `<version>` placeholder with the version number included in the filename of the Docker image you've just downloaded:
+
+```
+docker load -i /path/to/memgraph-<version>-docker.tar.gz
+```
+
+<p id="run-memgraph-server">Now, start the Memgraph server by issuing this command:</p>
+
+```
+docker run -p 7687:7687 -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph memgraph
+```
+
+If successful, you should be greeted with the following message:
+
+```
+Starting 2 BoltS workers
+BoltS server is fully armed and operational
+BoltS listening on 0.0.0.0:7687
+```
+
+The server is now ready to process your queries.
+
+If you want to stop the server, press `Ctrl + C` or `Cmd + .`. If you do, make sure you run it back again using the previous command. The server needs to be running to execute the next step.
+
+You now have Memgraph installed and running. The next step is to play around by executing some queries against it using the CLI (command-line interface) tool.
+
+## Step 2 — Querying Memgraph Through the CLI
+
+Memgraph comes with a CLI tool named `mg_client`. It allows you to execute Cypher queries against the Memgraph server using the command line.
```

Comment:
Please use https://github.com/memgraph/mgconsole. Maybe that's too much work because it doesn't work natively on macOS.

Reply:
As you mentioned it's too complex on max for now.